packages/core/graph/src/AdjacencyList.js

```diff
@@ -142,21 +164,23 @@ export default class AdjacencyList<TEdgeType: number = 1> {
       distribution += (bucket.size * (bucket.size + 1)) / 2;
     }
 
+    // TODO: Figure out if this calculation is still correct...
```

Comment:
Is this still TODO? Or did you forget to remove?

Reply:
Nice catch! I replaced it with some documented code that hopefully explains how it is (still) correct.